backend/routes/domains.js

```diff
@@ -21,33 +21,61 @@ module.exports = function (router) {
 
       log.debug('Trying to verify DNS records of domain', domain, hostname)
 
+      // Verify the domain is RFC-valid
+      if (!domain.match(DNS_VALID)) {
+        log.debug(`${domain} is not a valid domain`)
+        return res.send({
+          success: true,
```

Comment:
should this be success:false? same comment for similar code below.

Reply:
I'm not trying to alter outputs from what the FE expects here.

Reply:
+1 for a standardized output in general, however.